<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolChangeVolume_gui</class>
 <widget class="QWidget" name="ToolChangeVolume_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="radioAdd">
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinAdd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> dB</string>
     </property>
     <property name="minimum">
      <double>-200.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioMultiply">
     <property name="text">
      <string>Multiplier</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinMultiply">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="radioNormalize">
     <property name="text">
      <string>Normaliser</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinNormalize">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
